src/utils/metrics/prometheus.go

```diff
@@ -210,7 +211,7 @@ var PushGatewayCA string
 func getTLSConfig() (*tls.Config, error) {
 	rootCAs, err := x509.SystemCertPool()
 	if err != nil {
-		log.Println("Can't get system cert pool")
+		return nil, fmt.Errorf("can't get system cert pool: %w", err)
```

Comment:
I'll probably cherry-pick this change into main since it's a bug that has to be fixed asap but I can't merge the whole PR because of other comments

Reply:
hm, I saw that this problem was solved in fresh go1.18 for windows )) https://go.dev/doc/go1.18#crypto/x509
And I made it incorrectly. Better create empty cert pool instead of error because there is enough to create empty pool
In another project we do next:
```
var roots *x509.CertPool
	var err error
	// use system pool as default
	if roots, err = x509.SystemCertPool(); err != nil {
		log.WithError(err).WithField(logging.FieldKeyEventCode, logging.EventCodeErrorGeneral).
			Errorln("Can't load system ca certificates")
	}
	if roots == nil {
		roots = x509.NewCertPool()
	}
```

Reply:
@Lagovas good point, also thought about that. Unfortunately I'll have to wait with switching to go1.18 as not all the tooling I use support it yet (but I'll do the empty pool creation still)

Reply:
keep in mind that empty cert pool will works with our gates that use self-signed certificate but will fail for domain-based gates' URLs, because of need system cert pool with global known CA certs. but it should not panic, just pushing metrics will not work

Reply:
I understand, it's still better than not working metrics on windows even for IP-based gateways